client/components/public/FAQ.jsx

```diff
@@ -41,19 +43,11 @@ const gearPricing = (
     </ul> */}
     <ul>
       <li>Prices on varying styles range from $20&ndash;$35, additional $2 for extended sizes. Gear store open {regularRegistrationStart}&ndash;{gearSaleEnd}.</li>
-      <li>Gear sales end midnight {gearSaleEnd}</li>
+      <li>Gear sales end midnight {gearSaleEnd}.</li>
       <li>Shirts will be ordered on Monday, April 17 and assuming no supply chain delays, should be shipped out or ready for pick-up by the end of the week of April 24.</li>
       <li>The sale of these shirts helps fund this event. Support the WWU Great Puzzle Hunt and wear our official Great Puzzle Hunt gear! Check out the styles, colors, and design.</li>
     </ul>
-    <p>
-      Our costs to you are kept low thanks to generous donations from our sponsors. Their contributions help cover costs of this event including web fees and development, graphics, prizes, advertising, and much more.
-    </p>
-    <p>
-      Please consider <a target="_blank" href="https://foundation.wwu.edu/greatpuzzlehunt">donating to the Great Puzzle Hunt</a>.
-    </p>
-    <p>
-    The WWU Great Puzzle Hunt operates under WWU Foundation's 501(c)(3) status, so all donations are tax deductible. 
-    </p>
+    <p> If you are able, please consider making a small donation (suggested $5 students, $10 nonstudents) and/or buying a shirt (our only fund raiser). Donations of any amount will help sustain this event. </p>
```

Comment:
should there be Donate and Gear links in this section?

Reply:
I think it would be a bit much. We've got a link to the gear page, which leads to cashnet and the donation option.